packages/core/helper-plugin/lib/src/components/RelationsInput/components/ShadowList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+
+const ShadowBox = styled(Box)`
+  position: relative;
+
+  &:before {
+    content: ${({ overflowing }) =>
+      overflowing === 'both' || overflowing === 'top' ? "''" : undefined};
```

Comment:
What do you think about re-naming this prop to `overflow`, to make it more imperative?